src/main/java/vention/customWebElement/WebElementImp.java

```diff
@@ -31,15 +31,23 @@ private WebElement waitElementToBeVisible(WebElement element) {
   }
 
   private WebElement waitElementToBeClickable(WebElement element, int seconds) {
-    return new WebDriverWait(DriverManager.getDriver(), Duration.ofSeconds(seconds)).until(ExpectedConditions.elementToBeClickable(element));
+    return new WebDriverWait(DriverManager.getDriver(), Duration.ofSeconds(seconds))
+        .until(ExpectedConditions.elementToBeClickable(element));
   }
 
   private WebElement waitElementToBeVisible(WebElement element, int seconds) {
-    return new WebDriverWait(DriverManager.getDriver(), Duration.ofSeconds(seconds)).until(ExpectedConditions.visibilityOf(element));
+    return new WebDriverWait(DriverManager.getDriver(), Duration.ofSeconds(seconds))
+        .until(ExpectedConditions.visibilityOf(element));
   }
 
   public static void clickByLocator(By locator, int seconds) {
-    new WebDriverWait(DriverManager.getDriver(), Duration.ofSeconds(seconds)).until(ExpectedConditions.elementToBeClickable(locator)).click();
+    new WebDriverWait(DriverManager.getDriver(), Duration.ofSeconds(seconds))
+        .until(ExpectedConditions.elementToBeClickable(locator)).click();
+  }
+
+  public static boolean waitForUrlContains(String partialUrl, int seconds) {
```

Comment:
It's not part of webelement logic, url is not related to any webelement. Let's move it to driver classes